### PR TITLE
Add KMIP CSPs + initial Seal Wrap list

### PR DESCRIPTION
**Pull request by cipherboy**
This doc's KMIP using EA and adding a similar section to the Seal Wrap docs. This list is not comprehensive but can be updated as people ask about _what_ is seal wrapped.

### CODE REVIEW

**Comment by lucymhdavies** [comment]
Thanks!

Should we also update https://www.vaultproject.io/docs/enterprise/fips/sealwrap ? That page already links to https://www.vaultproject.io/docs/enterprise/sealwrap, so perhaps duplicating the list is not required, but mentioning that a list exists and where to find it would be useful.

For visibility, this was added in Vault 1.7.0, just never documented:
https://github.com/hashicorp/vault/blob/main/CHANGELOG.md#170
> kmip (enterprise): Use entropy augmentation to generate kmip certificates

(think this means we'll need a backport to earlier doc versions too?)

**Comment by cipherboy** [comment]
> (think this means we'll need a backport to earlier doc versions too?)

Ah good catch. Only 1.9 is still supported now that 1.11 came out.

File: website/content/docs/enterprise/entropy-augmentation.mdx [hunk 1]
```diff
@@ -47,6 +47,8 @@ and include the following:
 - [PKI](/docs/secrets/pki) issuer key generation, but not for leaf certificate private keys
 - [`/sys/tools/random`](/api-docs/system/tools#generate-random-bytes) endpoint (Vault 1.11+)
 - [SSH](/docs/secrets/ssh) CA key generation, but not for key pair generation
+- [KMIP](/docs/secrets/kmip) uses EA for its TLS CA, server, and client
```
Comment (victorr) [inline website/content/docs/enterprise/entropy-augmentation.mdx:50]: Well, don't forget the "managed objects", which are the keys and secret data objects that KMIP manages for clients. I think this means that all storage in KMIP is sealed wrapped, but I am not familiar enough with the storage code to verify that that is true.

**Review by victorr — CHANGES_REQUESTED** [review]
Requesting changes as I think it is worthwhile mentioning the managed objects.

File: website/content/docs/enterprise/sealwrap.mdx [hunk 2]
```diff
@@ -72,6 +72,23 @@ In addition, it is possible to replicate from a Shamir-protected primary
...
+Seal Wrapping does incur some performance overhead.
+
+Some examples of places where seal wrapping is used include:
+
+- The [LDAP](/docs/auth/ldap), [RADIUS](/docs/auth/radius),
+  [Okta](/docs/auth/okta), and [AWS](/docs/auth/aws) auth methods,
+  for storing their config.
+- [PKI](/docs/secrets/pki) for storing the issuers and their keys,
+- [SSH](/docs/secrets/ssh) for storing the CA's keys,
+- [KMIP](/docs/secrets/kmip) uses EA for its TLS CA, server, and client
```
Comment (cipherboy) [inline website/content/docs/enterprise/sealwrap.mdx:88]: @victorr Ah, that change should be here. I don't think we use entropy augmentation for KMIP ManagedObjects right? We only seal wrap them?

Comment (victorr) [reply website/content/docs/enterprise/sealwrap.mdx:88]: Yes, my bad.

Comment (cipherboy) [reply website/content/docs/enterprise/sealwrap.mdx]: Thanks @victorr -- updated!

**Review by victorr — APPROVED** [review]
Thanks for taking care of this.

**Pull request merged**